docs/authentication/saml/overview.mdx

```diff
@@ -1,185 +1,81 @@
 ---
-title: SAML (Beta)
+title: SAML
```

Comment:
should it be "Enterprise SSO via SAML" ?

Reply:
I think yes